cypress/support/pages/course/pageImportCourseModal.js

```diff
@@ -0,0 +1,33 @@
+"use strict"
+
+class ImportCourseModal {
+
+	static #importShareCourseDialog = '[data-testid="import-modal"]'
+	static #importShareCourseToolsInfo = '[data-testid="import-modal-external-tools-info"]'
+	static #importShareCourseNameInput = '[data-testid="import-modal-name-input"]'
+	static #dialogConfirmButton = '[data-testid="dialog-confirm"]';
+
+	seeImportShareCourseDialogBox() {
+		cy.get(ImportCourseModal.#importShareCourseDialog).should("exist");
+	}
+
+	seeImportShareCourseToolsInfo() {
+		cy.get(ImportCourseModal.#importShareCourseToolsInfo).should("exist");
```

Comment:
same here with exists
and add tests for all texts like in the seeInfoTextInShareCourseDialog

- data-testid="dialog-title"
- data-testid="import-options-table-header"
- data-testid="import-options-personal-data-text"
- data-testid="import-modal-coursefiles-info"
- data-testid="import-modal-external-tools-protected-parameter-info"